README.md

```diff
@@ -1,13 +1,8 @@
 # The Swift Programming Language
 
-This repository contains a version of *The Swift Programming Language*
-that you can build using Swift-DocC.
-
-The version of *The Swift Programming Language* published on [docs.swift.org](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/)
-is currently built using a legacy toolchain,
-not from this repository or its content.
-The goal is to replace that version with a version built from this repository,
-working with the community to extend Swift-DocC where needed.
+This repository contains the source for *The Swift Programming Language*,
```

Comment:
```suggestion
This repository contains the source for *The Swift Programming Language* book (TSPL for short),
```

Seems like this would be a good place to introduce our short name for the book, since it appears in the source structure as `TSPL.docc`.

Reply:
Good to call this out. How about "sometimes abbreviated as TSPL"?